serve.ts

```diff
@@ -2,7 +2,7 @@ import esbuild from "esbuild";
 
 const init = async () => {
   const ctx = await esbuild.context({
-    entryPoints: ["static/scripts/app.ts", "static/scripts/audit.ts", "static/styles/app.css", "static/styles/audit.css"],
+    entryPoints: ["static/scripts/app.ts", "static/scripts/audit.ts", "static/styles/app.css", "static/styles/audit.css", "static/styles/toast.css"],
```

Comment:
no need to modify the `entryPoints` imported CSS files.

Reply:
@pavlovcik

Reply:
It needs to be built cause you tried to directly link it, instead of importing.